pyface/ui/qt4/tasks/advanced_editor_area_pane.py

```diff
@@ -575,7 +575,7 @@ def focusInEvent(self, event):
     # Signal handlers.
     # ------------------------------------------------------------------------
 
-    def _focus_changed(self, old, new):
+    def _focus_updated(self, old, new):
```

Comment:
these aren't trait change handlers, they are signal handlers used at the qt level, so the renaming is to avoid confusion.

Reply:
```suggestion
    def _update_active_editor(self, old, new):
```

Reply:
if this and the other suggestion are accepted, the corresponding uses of the signal handlers will also need to be updated